### PR TITLE
Update Audio Profiles to beta

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java [hunk 1]
```diff
@@ -134,17 +180,29 @@ public static void main(String... args) throws Exception {
...
             .defaultHelp(true)
-            .description("Synthesize a text or ssml.");
+            .description("Synthesize a text, text with audio effect profiles, or ssml.");
 
     MutuallyExclusiveGroup group = parser.addMutuallyExclusiveGroup().required(true);
-    group.addArgument("--text").help("The text file from which to synthesize speech.");
+    group
+        .addArgument("--text")
+        .help("The text file from which to synthesize speech.")
+        .nargs("+")
```
Comment (kurtisvg) [inline texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java]: This should probably be a different argument? `.addArgument("--effect-profile").required(false)` or something?

Comment (nnegrey) [reply texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java]: I think I'll just drop the CLI entirely like we did for Dialogflow

Comment (kurtisvg) [reply texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java]: 👍

File: texttospeech/cloud-client/README.md [hunk 2]
```diff
@@ -46,30 +46,8 @@ Synthesize text to an output audio file. [Java Code](https://github.com/GoogleCl
 mvn exec:java -DQuickstart
```
Comment (nirupa-kumar) [inline texttospeech/cloud-client/README.md]: Just one nit. Should we be removing this one too ?
```
mvn exec:java -DQuickstart
```

Comment (nnegrey) [reply texttospeech/cloud-client/README.md]: I don't think so, cus the quick start is still in a main() method entirely.

But I did totally delete the pom.xml profile for it, so I should put that back in.
I think it's nice that a quickstart is still runnable, but @kurtisvg thoughts?

Comment (kurtisvg) [reply texttospeech/cloud-client/README.md]: Is there anything mention in the Cloud Docs with regards to it? If so, I would leave it in. If not, it probably doesn't matter.

Comment (nnegrey) [reply texttospeech/cloud-client/README.md]: It does not. 
I'll remove it.

**Comment by nnegrey** [comment]
Friendly Ping @kurtisvg

**Pull request merged**